index.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <title>Portfolio mobile version</title>
+</head>
+<body>
+    <header>
+        <a href="#" class="logo">KGW</a>
+        <nav>
+         <img src="images/Menu.png" alt="menu">
+        </nav>
+    </header>
+    <section class="main">
+        <h1>Kidde George</h1>
+        <h2>Software Developer</h2>
+        <p>I can help you build a product, feature or website Look through some of my work and experience! If you like what you see and have a project you need coded,  don't hestiate to contact me.</p>
+    </section>
+    <section class="home">
+        <hr>
+        <div class="technologies">
+            <ul>
+                <li><h3>Language</h3>
+                </li>
+                <li>JavaScript</li>
+                <li>Ruby</li>
+                <li>Html</li>
+                <li>Css</li>
+            </ul>
+            <ul>
+                <li><h3>Frameworks</h3></li>
+                <li>Bootstrap</li>
+                <li>Ruby on Rails</li>
+                <li>RSPec</li>
+                <li>CapyBara</li>
+                <li>Selenium</li>
+            </ul>
+            <ul>
+                <li><h3>Skills</h3></li>
+                <li>Codekit</li>
+                <li>Github</li>
+                <li>Codepen</li>
+                <li>Gitlab</li>
+                <li>Terminal</li>
+            </ul>
+        </div>
+        <hr>
+        <div class="icons">
+            <a href="#"> <img src="images/Vector.png" alt="Github"> </a>
+            <a href="#"> <img src="images/Icon-Linkedin.png" alt="LinkedIn"> </a>
+            <a href="#"> <img src="images/angelist-1.png" alt="Angelist"> </a>
+            <a href="#"> <img src="images/Icon-Twitter.png" alt="Twitter"> </a>
+            <a href="#"> <img src="images/Icon-Medium.png" alt="Medium"> </a>
+        </div>
```

Comment:
- [ ] Kindy use the `ul` tag to group your icons, and li per icon, as opposed to using `div` tag this is a project requirement.